examples/master.js

```diff
@@ -10,28 +10,32 @@
 
 var Discover = require("../");
 
-var d = new Discover({ key : process.argv[2], weight : Date.now() * -1, mastersRequired : 2 });
+var d = new Discover({ 
+	key : process.argv[2], weight : Date.now() * -1
+	, mastersRequired : 2
+	//, leadershipElector : Discover.NoLeadershipElection
+});
 
 console.log('I am ' + d.broadcast.instanceUuid);
 
 d.on("added", function (obj) {
-	console.log("New node discovered on the network.");
+	process.stdout.write("\nNew node discovered on the network.");
```

Comment:
Why was these changed? I know it doesn't impact the library, just curious.

Reply:
I use this example to test the behavior. The output bothered me because there were too many blank lines and it didn't feel like a repl.